pkgs/build-support/vm/default.nix

```
@@ -258,7 +257,7 @@ rec {
     # Write the command to start the VM to a file so that the user can
     # debug inside the VM if the build fails (when Nix is called with
     # the -K option to preserve the temporary build directory).
-    cat > ./run-vm <<EOF
+    ${coreutils}/bin/cat > ./run-vm <<EOF
     #! ${bash}/bin/sh
```

Unrelated to this PR (doesn't need to be addressed):

```suggestion
    #! ${runtimeShell}
```